addons/l10n_jo_edi/models/res_config_settings.py

```diff
@@ -5,6 +5,6 @@ class ResConfigSettings(models.TransientModel):
     _inherit = 'res.config.settings'
 
     l10n_jo_edi_sequence_income_source = fields.Char(string="Sequence of Income Source", related="company_id.l10n_jo_edi_sequence_income_source", readonly=False)
-    l10n_jo_edi_secret_key = fields.Char(string="JoFotara Secret Key", related="company_id.l10n_jo_edi_secret_key", readonly=False)
-    l10n_jo_edi_client_identifier = fields.Char(string="JoFotara Client ID", related="company_id.l10n_jo_edi_client_identifier", readonly=False)
-    l10n_jo_edi_taxpayer_type = fields.Selection(string="Taxpayer type", related="company_id.l10n_jo_edi_taxpayer_type", readonly=False)
+    l10n_jo_edi_secret_key = fields.Char(string="Secret Key", related="company_id.l10n_jo_edi_secret_key", readonly=False)
```

Comment:
@ygu-odoo The problem with this Client ID stuff is that other localizations also have Client ID somewhere, why it is a good idea to put JoFotara here. I think, if you really do not want it, we might override it in the view still with string="Client ID".